Objects/Converters/ConverterAutocadCivil/ConverterAdvanceSteel2023/ConverterAdvanceSteel2023.csproj

```diff
@@ -26,7 +26,6 @@
 
 	<Target Name="PostBuild" AfterTargets="PostBuildEvent" Condition="'$(IsDesktopBuild)' == true">
```

Comment:
We've moved away from having a "copy/paste" approach to the post build events in converters to use a specific post build target shared by all projects (it lives in the Directory.Build.targets file at the repo root)

In order to get the converter `dll` copied, you now only need to add a property `<CopyToKitFolder>true</CopyToKitFolder>`

See the revit converter 👇🏼
https://github.com/specklesystems/speckle-sharp/blob/eb68acd457ba3a9d1512baab28d93230ba1ba086/Objects/Converters/ConverterRevit/ConverterRevit2023/ConverterRevit2023.csproj#L11


For any other dependencies, you should specify them in a target that runs after `CopyToKitFolder` does. Revit converter already does this, see here 👇🏼

https://github.com/specklesystems/speckle-sharp/blob/eb68acd457ba3a9d1512baab28d93230ba1ba086/Objects/Converters/ConverterRevit/ConverterRevit2023/ConverterRevit2023.csproj#L32-L36

Reply:
I did it.